vite.config.ts

```diff
@@ -0,0 +1,15 @@
+import process from 'node:process';
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import viteTsconfigPaths from 'vite-tsconfig-paths';
+import svgrPlugin from 'vite-plugin-svgr';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react(), viteTsconfigPaths(), svgrPlugin()],
+  base: process.env.PATH_PREFIX || '/',
+  build: {
```

Comment:
Considering loading data files with dynamic import or http request, in this way we can split the files from the bundle

Reply:
Right👍, using asynchronous loading for data files can improve user experience. Skeleton screens can also be considered where appropriate.

Regarding the 6MB index.js file mentioned previously, the main concern is that after build completion, the large bundle size is unfavorable for initial page load. That's why splitting it into separate chunks during build was suggested. This could be optimized later.